### PR TITLE
Desktop version creation

### PR DESCRIPTION
**Pull request by elmar8287**
- Replace the screenshot for the desktop version
- Make changes in the CSS file

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @elmar8287, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters errors ✔️  

## Aspects to improve ♻️

- [x] Please, for each project divide the info in 2 columns

| original | your project |
| --- | --- |
| image | image |
| image | image |




- [x] In all CSS stylesheet, I don't see a media query for the Desktop breakpoint file.
- Hint: `@media screen and (max-width: 1080px) {`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL] The background-images size is too big. I kindly invite you to fix it to be according to the Figma design.

| Original | your project |
| --- | --- |
| image | image |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Dear @vichuge,

Thank you for your review! 👍

I have fixed all you marked. Please, check again.

My regards,

Elmar

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 > We build our portfolio mobile website
 
-![Screen Shot](./scrnsht.PNG)
+![Screen Shot](./desktop-version.PNG)
 
 This is the first milestone in the process of creating our portfolio website.
```
Comment (ElijahTobs) [inline README.md:10]: - [ ] _[Optional]_ This milestone is for the desktop version but your README title and description is for the mobile version. Kindly fix that

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @elmar8287, 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

_Check for changes marked as Optional in the inline comments below._


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**